### DISCUSSION TITLE
How to enable "save to pocket" extensions?

### DISCUSSION THREAD

**Discussion by eMUQI** [zen-browser/desktop#417] [Q&A] [answered]
I understand that some people don't like the default installation of the Pocket extension. But how can i enable the Pocket plugin on Zen Browser?

I tried following this tutorial, setting `extensions.pocket.enabled` to `true`, but I still can't get the Pocket extension to work.
https://support.mozilla.org/en-US/kb/disable-or-re-enable-pocket-for-firefox

**Comment by eMUQI** [comment]
Is this a bug that wasn't anticipated? Should I report it as an issue?

**Comment by eMUQI** [comment]
> I tried following this tutorial, setting `extensions.pocket.enabled` to `true`, but I still can't get the Pocket extension to work. https://support.mozilla.org/en-US/kb/disable-or-re-enable-pocket-for-firefox

Is this a bug that wasn't anticipated? Should I report it as an issue?

**Comment by eMUQI** [comment, marked as answer]
I found the solution. Even after turning on the plugin, I have to manually add it to the toolbar.

![image](https://github.com/user-attachments/assets/00f39c71-d133-4a57-a139-879daac7eb5f)

**Comment by nuf0n** [comment]
Thanks for the lifesaving solution, @eMUQI!

For the devs: Perhaps this step could be added to the documentation?

**Comment by dexterque** [comment]
有用！👍

**Comment by dexterque** [comment]
Sadly! I found the Pocket is closing down...
